### DISCUSSION TITLE
Rename the main file to `mod.ts` on Deno Land

### DISCUSSION THREAD

**Discussion by rojvv** [panva/jose#461] [Ideas]
`https://deno.land/x/jose/mod.ts`

**Comment by panva** [comment]
Hi @roj1512, 

What would be the point of this change?

**Reply by rojvv** [reply]
Consistency

**Reply by rojvv** [reply]
Everyone does that. https://deno.land/manual@v1.26.2/contributing/style_guide#do-not-use-the-filename-indextsindexjs

**Reply by panva** [reply]
That doc is probably outdated, `index.*` is [already treated](https://deno.land/x/jose@v4.10.0?doc) as a valid module input.

<img width="1239" alt="Screenshot 2022-10-18 at 16 24 17" src="https://user-images.githubusercontent.com/241506/196458150-b3f11b96-d68d-4602-8088-ed994fd0370d.png">

I will consider this for the next major release though.

**Reply by panva** [reply]
Also, right at the top of the doc you linked there's a disclaimer.

> ⚠️ Note that this is the style guide for **internal runtime code** in the Deno runtime, and in the Deno standard library. This is not meant as a general style guide for users of Deno.

**Reply by rojvv** [reply]
What I sent is not doc, it is Deno's STD style guide. Although, thanks for "considering it for the next major release"!

**Reply by rojvv** [reply]
Yes, I know about that disclaimer. I just sent that to show you that `mod.ts` is much more common in the Deno world.
